backend-api/src/functions/asyncIssueBiometricCredential/asyncIssueBiometricCredentialHandler.ts

```diff
@@ -55,7 +55,7 @@ export async function lambdaHandlerConstructor(
   event: SQSEvent,
   context: Context,
 ): Promise<void> {
-  setupLogger(context);
+  setupLogger(context, /*userAgent=*/ undefined);
```

Comment:
suggestions:

Option 1 (fine but not my pref):

```suggestion
  setupLogger(context, undefined);
```

Option 2:

Update `setupLogger` `userAgent` param to be an optional `string` rather than `string | undefined`:

```ts
export const setupLogger = (
  context: Context,
  userAgentHeader?: string
) => {
```

So when you call the function in this lambda (or any other lambda where we don't have `req.headers` you can just do:

```suggestion
  setupLogger(context)
```